frontend/webapp/components/main/header/index.tsx

```diff
@@ -3,7 +3,7 @@ import Image from 'next/image';
 import styled from 'styled-components';
 import { PlatformTitle } from './cp-title';
 import { useConnectionStore } from '@/store';
-import { Status } from '@/reuseable-components';
+import { ConnectionStatus } from '@/reuseable-components';
 import { NotificationManager } from '@/components/notification';
 
 interface MainHeaderProps {}
```

Comment:
we can now use our new common style to make this component smaller:


export const HeaderContainer = styled(FlexRow)`
  width: 100%;
  padding: 12px 0;
  background-color: ${({ theme }) => theme.colors.darker_grey};
  border-bottom: 1px solid rgba(249, 249, 249, 0.16);
`;

export const HeaderSection = styled(FlexRow)`
  margin: 0 32px;
  flex: 1;
  &:first-child {
    margin-right: auto;
  }
  &:last-child {
    margin-left: auto;
  }
`;